charger/amtron.go

```diff
@@ -91,18 +91,17 @@ func (wb *Amtron) Status() (api.ChargeStatus, error) {
 		return api.StatusNone, err
 	}
 
-	switch b[0] {
+	switch b[1] {
 	case 1, 2:
 		return api.StatusA, nil
-	case 3, 4:
+	case 3, 4, 5:
 		return api.StatusB, nil
-	case 5, 6:
-		// TODO check C1 -> B?
+	case 6:
```

Comment:
```suggestion
	case 5, 6:
```

Reply:
Wie haben bei alfen auch c1=connected=b gemappt?

Reply:
Alfen:

	switch r := rune(b[0]); r {
	case 'A', 'B', 'D', 'E', 'F':
		return api.ChargeStatus(r), nil
	case 'C':
		// C1 is "connected"
		if rune(b[1]) == '1' {
			return api.StatusB, nil
		}
		return api.StatusC, nil

Reply:
Vermutlich ist dann dann dort auch falsch.